requirements/prod.txt

```diff
@@ -0,0 +1,146 @@
+#
+# This file is autogenerated by pip-compile
+# To update, run:
+#
+#    pip-compile --allow-unsafe requirements/prod.in
+#
+appdirs==1.4.4
+    # via fs
+authlib==0.15.3
+    # via simple-salesforce
+certifi==2020.12.5
+    # via requests
+cffi==1.14.5
+    # via cryptography
+chardet==4.0.0
+    # via requests
+click==7.1.2
+    # via
+    #   -r requirements/prod.in
+    #   snowfakery
+coloredlogs==15.0
+    # via -r requirements/prod.in
+cryptography==3.4.7
+    # via
+    #   -r requirements/prod.in
+    #   authlib
+    #   jwcrypto
+faker==6.6.3
+    # via
+    #   -r requirements/prod.in
+    #   snowfakery
+fs==2.4.12
+    # via -r requirements/prod.in
+github3.py==2.0.0
+    # via -r requirements/prod.in
+gvgen==1.0
+    # via snowfakery
+humanfriendly==9.1
+    # via coloredlogs
+idna==2.10
+    # via requests
+importlib-metadata==3.7.3
+    # via keyring
+jinja2==2.11.3
+    # via
+    #   -r requirements/prod.in
+    #   snowfakery
+jwcrypto==0.8
+    # via github3.py
+keyring==23.0.1
+    # via -r requirements/prod.in
+lxml==4.6.3
+    # via -r requirements/prod.in
+markupsafe==1.1.1
+    # via jinja2
+pycparser==2.20
+    # via cffi
+pydantic==1.8.1
+    # via -r requirements/prod.in
+pyjwt==2.0.1
+    # via -r requirements/prod.in
+python-dateutil==2.8.1
+    # via
+    #   faker
+    #   github3.py
+    #   snowfakery
+pytz==2021.1
+    # via
+    #   -r requirements/prod.in
+    #   fs
+pyyaml==5.4.1
+    # via snowfakery
+requests-futures==1.0.0
+    # via -r requirements/prod.in
+requests==2.25.1
+    # via
+    #   -r requirements/prod.in
+    #   github3.py
+    #   requests-futures
+    #   robotframework-requests
+    #   salesforce-bulk
+    #   simple-salesforce
+robotframework-lint==1.1
+    # via -r requirements/prod.in
+robotframework-pabot==1.11
+    # via -r requirements/prod.in
+robotframework-pythonlibcore==2.2.1
+    # via robotframework-seleniumlibrary
+robotframework-requests==0.8.1
+    # via -r requirements/prod.in
+robotframework-seleniumlibrary==5.1.3
```

Comment:
We shouldn't be updating robotframework-seleniumlibrary to 5.x until Bryan has a look.